src/keymap.cpp

```diff
@@ -743,7 +743,7 @@ static const std::vector<OpDesc> opdescs = {
 	{
 		OP_INT_GOTO_URL,
 		"gotourl",
-		KeyCombination("internal-goto-url"),
+		{},
```

Comment:
Also replace `KeyCombination("internal-set")` above (can't comment on it directly due to github comment restriction)

Reply:
Done